src/angular/src/app/pages/settings/integrations.component.html

```
@@ -0,0 +1,123 @@
+    }
+  </h3>
+  <div class="restart-note">Attach instances to path pairs to control which downloads notify which *arr.</div>
+  @if (errorMessage) {
+    <div class="error-message" role="alert">{{ errorMessage }}</div>
+  }
+
+  @if (adding) {
+    <div class="instance-form">
+      <ng-container *ngTemplateOutlet="instanceFormFields; context: { form: addForm }"></ng-container>
+      <div class="form-actions">
+        <button class="btn btn-sm btn-save" (click)="onSaveAdd()">Save</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)